workflow/scripts/compare_diffexp.py

```diff
@@ -50,7 +50,15 @@ def prepare(df):
     combined.with_columns(
         (-pl.col("pval_min").log(base=10) * pl.col("difference")).alias("pi_value")
     )
-    .sort(pl.col("pi_value").abs(), descending=True)
+    .with_columns(
+        pl.col("pi_value").fill_nan(None),
+        pl.col("difference").fill_nan(None),
+    )
```

Comment:
This seems convoluted, but at least it's well documented:
https://docs.pola.rs/user-guide/expressions/missing-data/#notanumber-or-nan-values